Source/WebCore/html/shadow/SliderThumbElement.cpp

```diff
@@ -555,28 +548,10 @@ std::optional<Style::ElementStyle> SliderThumbElement::resolveCustomStyle(const
     if (!hostStyle)
         return std::nullopt;
 
-    switch (hostStyle->effectiveAppearance()) {
-    case MediaSliderPart:
-    case MediaVolumeSliderPart:
-    case MediaFullScreenVolumeSliderPart:
-        m_shadowPseudoId = ShadowPseudoIds::webkitMediaSliderThumb();
-        break;
-    default:
-        m_shadowPseudoId = ShadowPseudoIds::webkitSliderThumb();
-        break;
-    }
+    setPseudo(ShadowPseudoIds::webkitSliderThumb());
```

Comment:
Can you call this in SliderThumbElement::create() ? like https://webkit-search.igalia.com/webkit/rev/6852fe37c6f7616408223dea6a3ee7fd890a17d4/Source/WebCore/html/shadow/SpinButtonElement.cpp#62-67